lib/glucose-get-last.js

```diff
@@ -2,7 +2,9 @@ var getLastGlucose = function (data) {
     data = data.map(function prepGlucose (obj) {
         //Support the NS sgv field to avoid having to convert in a custom way
         obj.glucose = obj.glucose || obj.sgv;
-        return obj;
+        if ( obj.glucose !== null ) {
+            return obj;
+        }
```

Comment:
This will cause the map to return `undefined` instead of `null` values. If we first filter out those `null` values (via #1328) we can end up with a more concise array with only relevant values (no `null` or `undefined`).

What do you think?

Reply:
Ok, merged that. Null checks are still good, though, so any reason not to merge this too?